### PR TITLE
Kaira Johnson Lab Tic Tac Toe

### PR DESCRIPTION
**Pull request by kairajohnson**
completeness: 3.5
comfort: 3.5

### CODE REVIEW

**Comment by kairajohnson** [comment]
I made an if statement for my array of divs to display yellow if the defined indexes are a match. can you advise me on how to approach this

```
var winningBoxes=[
[box.eq("0"),box.eq("1"),box.eq("2")]]
var reset;

$(".box").toArray()

for (var i=0; i < 1; i++) {
$(".box").click(function() {
if (counter%2===0) {
  $(this).css("background-color", "blue")
 $(this).html("x")
 counter = counter + 1;
**if ($(this)===winningBoxes){
   $(this).css("background-color", "yellow")
 }**
}
```

File: script.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+$(".box").toArray()
+
+for (var i=0; i < 1; i++) {
+$(".box").click(function() {
+if (counter%2===0) {
+  $(this).css("background-color", "blue")
+ $(this).html("x")
+ counter = counter + 1;
+if ($(this)===winningBoxes){
```
Comment (joe-gz) [inline script.js:24]: Why nest this if statement within here?
right now `$(this)` is only going to register whatever box you click on, that wouldn't equate to the array of the winning row at any point

File: script.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+ var box = $(".box");
+ var activeBox = $(".box").eq("")
+var counter= 0;
+var winningBoxes=[
+[box.eq("0"),box.eq("1"),box.eq("2")],
```
Comment (joe-gz) [inline script.js:5]: Right now this only has a combination of the top row, is that hardcoded as a test?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+// [0,4,8],
+// [6,4,2]
+]
+var reset;
+
+$(".box").toArray()
+
+for (var i=0; i < 1; i++) {
+$(".box").click(function() {
+if (counter%2===0) {
```
Comment (joe-gz) [inline script.js:20]: this is great!

File: script.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+ var box = $(".box");
+ var activeBox = $(".box").eq("")
```
Comment (joe-gz) [inline script.js:2]: can this be removed?

**Comment by joe-gz** [comment]
:+1: This is a good start Kaira! I added in some notes, especially regarding your question above. Also, please be aware of indenting in your code, there are definitely some spots in the JS that can be cleaned up

**Comment by kairajohnson** [comment]
Thanks Joe. I just finished my quiz and didn't get as far as I wanted in
Pixel Art. Can I schedule one on one time with you tomorrow? I don't know
if its too much to ask but walking through it with me will be a big help. I
get stuck in the very beginning and draw a blank. I have an idea of what to
do but when I go to execute it, I go back to zero fast This seems to be my
issue with every homework assignment:/ My hope is that meeting with you and
going step by step will help me "get it" so I can move forward.

thanks

On Sun, Mar 6, 2016 at 10:32 PM, Joe Glatman Zaretsky <
notifications@github.com> wrote:

> [image: :+1:] This is a good start Kaira! I added in some notes,
> especially regarding your question above. Also, please be aware of
> indenting in your code, there are definitely some spots in the JS that can
> be cleaned up
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ga-wdi-exercises/tic_tac_toe/pull/91#issuecomment-193074682
> .

## 

*LOVE *

_NOTE: This e-mail transmission, including any attachments, is intended
only for the named recipient(s) and may contain information that is
privileged, confidential and/or exempt from disclosure under applicable
law. If you have received this email by intent or accident, you are
forbidden to disclose any information attached or written with anyone
outside of whom this email was intended for. If you have received this
transmission in error, or are not the named recipient(s), please notify
Kaira Johnson kjohnson1913@gmail.com immediately by return e-mail and
permanently delete this transmission, including any attachments._

**Pull request closed without merging**